lib/media.js

```diff
@@ -71,8 +71,8 @@ function combineFileContents(fileArray, directory, filePrefix) {
     .then(function (fileContents) {
       const joined = fileContents.join('');
 
-      if (util.getByteLength(joined) > 50000) {
-        log('warn', 'Combined CSS contents are beyond the 50000 byte limit specified by AMPHTML.');
+      if (util.getByteLength(joined) > 75000) {
```

Comment:
Might be good to link to the relevant docs to show where this number comes from.